spec/iex/endpoints/ref_data_spec.rb

```diff
@@ -3,14 +3,28 @@
 describe IEX::Api::Client do
   include_context 'client'
 
-  describe 'ISIN Mapping', vcr: { cassette_name: 'ref-data/isin' } do
-    subject do
-      client.post('ref-data/isin', isin: ['US0378331005'], token: client.secret_token)
-    end
-    it 'retrieves a ticker by ISIN' do
+  describe '#ref_data_isin', vcr: { cassette_name: 'ref-data/isin' } do
+    subject { client.ref_data_isin(['US0378331005']) }
+
+    it 'converts ISIN to IEX Cloud symbols' do
       expect(subject.count).to eq(2)
-      expect(subject.first).to eq('exchange' => 'NAS', 'iexId' => 'IEX_4D48333344362D52', 'region' => 'US', 'symbol' => 'AAPL')
-      expect(subject.last).to eq('exchange' => 'ETR', 'iexId' => 'IEX_464D46474C312D52', 'region' => 'DE', 'symbol' => 'APC-GY')
+      expect(subject.first).to eq('exchange' => 'NAS', 'iex_id' => 'IEX_4D48333344362D52', 'region' => 'US', 'symbol' => 'AAPL')
+      expect(subject.last).to eq('exchange' => 'ETR', 'iex_id' => 'IEX_464D46474C312D52', 'region' => 'DE', 'symbol' => 'APC-GY')
+    end
+
+    context 'with mapped option', vcr: { cassette_name: 'ref-data/isin_mapped' } do
```

Comment:
This context does not depend/inherit the parent one. Split them up here so each has its own section.